Tasks/PackerBuild/src/TemplateFileProviderBase.ts

```diff
@@ -22,20 +22,53 @@ export default class TemplateFileProviderBase {
         tl.debug("template location: " + tempFileLocation);
     }
 
-    public updateTemplateFile(content: string): void {
+    public readTemplateFileJson() {
+        var content = utils.readJsonFile(this._templateFileLocation);
+        var templateJson = null;
+
+        try {
+            templateJson = JSON.parse(content);
+        } catch (err) {
```

Comment:
yes,earlier i was thinking of ignoring the error and proceeding, but it seems better to just fail here.